server/routes.js

```diff
@@ -217,7 +217,7 @@ export default async app => {
   app.post('/webhooks/stripe', stripeWebhook); // when it gets a new subscription invoice
   app.post('/webhooks/transferwise', transferwiseWebhook); // when it gets a new subscription invoice
   app.post('/webhooks/privacy', privacyWebhook); // when it gets a new subscription invoice
-  app.post('/webhooks/paypal', paypalWebhook);
+  app.post('/webhooks/paypal/:hostId?', paypalWebhook);
```

Comment:
Nice solution.

Reply:
It was necessary for the refund part because the refund payload doesn't have any info about which transaction/order/host we should look at. But ultimately, this can help us optimize other endpoints as we'll only need to look at the orders/transactions that are related to that host